Library/Formula/b2-tools.rb

```diff
@@ -13,12 +13,14 @@ class B2Tools < Formula
 
   def install
     ENV.prepend_create_path "PYTHONPATH", libexec+"lib/python2.7/site-packages"
+
+    system "python", "setup.py", "install", "--prefix=#{libexec}"
```

Comment:
The error there is:
```
pkg_resources.DistributionNotFound: The 'six>=1.10.0' distribution was not found and is required by b2
```
The formula is missing the necessary resources to run as-expected, apparently. If you use [homebrew-pypi-poet](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Python-for-Formula-Authors.md#dependencies) it'll do the hard work for you on that and give you a nice list to just copy&paste.